### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by ablaom**
- #958
- Add a link to "Machine Learning Property Loans for Fun and Profit" blogpost

### CODE REVIEW

**Comment by ablaom** [comment]
There is a problem with the docs - EvoTrees has renamed it's learning rate hyperparameter to `eta`, which is a problem in `docs/src/controlling_iterative_models.md`.

Working on a fix.
